commercetools-models/src/main/java/io/sphere/sdk/productdiscounts/ProductDiscountDraftBuilder.java

```diff
@@ -0,0 +1,71 @@
+package io.sphere.sdk.productdiscounts;
+
+import javax.annotation.Nullable;
+
+import io.sphere.sdk.models.Base;
+import io.sphere.sdk.models.Builder;
+import io.sphere.sdk.models.LocalizedString;
+
+public class ProductDiscountDraftBuilder extends Base implements Builder<ProductDiscountDraft> {
```

Comment:
can you make the class final, please? The constructor is private so subclasses are not possible.